ush/rocoto/setup_workflow.py

```diff
@@ -397,6 +397,10 @@ def get_gdasgfs_tasks(dict_configs, cdump='gdas'):
     envars.append(rocoto.create_envar(name='CDUMP', value=f'{cdump}'))
     envars.append(rocoto.create_envar(name='PDY', value='<cyclestr>@Y@m@d</cyclestr>'))
     envars.append(rocoto.create_envar(name='cyc', value='<cyclestr>@H</cyclestr>'))
+    envars.append(rocoto.create_envar(name='GDATE', value='<cyclestr offset="-6:00:00">@Y@m@d@H</cyclestr>'))
+    envars.append(rocoto.create_envar(name='GDUMP', value='gdas'))
+    envars.append(rocoto.create_envar(name='gPDY', value='<cyclestr offset="-6:00:00">@Y@m@d</cyclestr>'))
+    envars.append(rocoto.create_envar(name='gcyc', value='<cyclestr offset="-6:00:00">@H</cyclestr>'))
```

Comment:
Do all jobs now need these variables?

Reply:
Any jobs that use COMIN_GES_OBS need these, so many of the GSI/EnKF jobs. Not all jobs need them but I didn't want to set a job list in the setup scripts to only add them for some. Per a discussion with Russ about COMIN_OBS/COMIN_GES_OBS this is just a patch for dev users running the WCOSS2 ops version. This patch could go away once efforts are taken to investigate using compath.py more widely and remove the existing emc mode workarounds, like this.

Reply:
@RussTreadon-NOAA Agreed, will be looking to rework this during the dev side of this port (next few months).